contracts/MirrorNodeFFI.sol

```diff
@@ -101,6 +101,16 @@ contract MirrorNodeFFI is MirrorNode {
         ));
     }
 
+    /**
```

Comment:
Let's just disable Slither for the offending line

```solidity
    // State is initialized using `vm.store` to avoid `EvmError: StateChangeDuringStaticCall`
    // slither-disable-next-line uninitialized-state
    mapping (string endpoint => string response) private _responses;
```